cmake/EthCompilerSettings.cmake

```diff
@@ -200,6 +200,8 @@ if (SANITIZE)
 	elseif (sanitizer STREQUAL "undefined")
 		# The following flags not used by fuzzer but used by us may create problems, so consider
 		# disabling them: alignment, pointer-overflow.
+		# The following flag is not used by us to reduce terminal noise
+		# i.e., warnings printed on stderr: unsigned-integer-overflow
```

Comment:
> The C++ standard does not officially consider unsigned integer overflows to be undefined behavior since it is implementation independent.

I think it would be good to include this extra bit of info here.

Reply:
Added a comment.